pybossa/view/projects.py

```diff
@@ -3169,6 +3168,8 @@ def ext_config(short_name):
                     flash(gettext('Please correct the errors', 'error'))
                 ext_conf[form_name] = form.data
                 ext_conf[form_name].pop('csrf_token', None)     #Fflask-wtf v0.14.2 issue 102
+                if not ext_conf[form_name].get('bpv_key_id'):
```

Comment:
Hardcoding an instance specific parameter here. Is there a way to make this more generic and prevent the same issue for other fields in the future?

Reply:
Yes I think it's doable, let me investigate a little bit.